api/api.yaml

```diff
@@ -0,0 +1,61 @@
+swagger: "2.0"
```

Comment:
https://github.com/kudobuzz/api-standards
Do take a look at the api standard

Reply:
yes i wrote it in swagger first to better understand documenting api's will work on making it open api 3.0 today